ci-operator/step-registry/cucushift/e2e/cucushift-e2e-commands.sh

```diff
@@ -16,6 +16,11 @@ if [ -f "${SHARED_DIR}/proxy-conf.sh" ] ; then
 fi
 
 export E2E_RUN_TAGS="${E2E_RUN_TAGS} and ${TAG_VERSION}"
+if [ -z "${E2E_SKIP_TAGS}" ] ; then
+    export E2E_SKIP_TAGS="not @customer and not @security"
+else
+    export E2E_SKIP_TAGS="${E2E_SKIP_TAGS} and not @customer and not @security"
+fi
```

Comment:
@liangxia I see we already updated the env var in ref.yaml, why do we have to update the commands.sh?

Reply:
Those env vars can be overwrite by specific jobs, add it there in case new jobs added with those env vars overwrite but does not add those tags, then we will got another leak.

Reply:
OK thanks. If `E2E_SKIP_TAGS` already contains the string `not @customer and not @security`, then `export E2E_SKIP_TAGS="${E2E_SKIP_TAGS} and not @customer and not @security"` will contain duplicate tag strings. But I guess this is generally fine.